### PR TITLE
[14.0] [FIX] sale_partner_approval: Fixed Can Sale is lazy updated

### PR DESCRIPTION
**Pull request by Nikul-OSI**
@dreispt Fixed Can Sell is lazy updated.

### CODE REVIEW

**Review by dreispt — APPROVED** [review]
Reviewed, tested, and fixed tests

File: sale_partner_approval/data/init_sale_ok.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<odoo noupdate="1">
```
Comment (flotho) [inline sale_partner_approval/data/init_sale_ok.xml:1]: FMI, wont it be better to use pre init hook? is it still recommended ?

Comment (dreispt) [reply sale_partner_approval/data/init_sale_ok.xml:1]: This should be equivalent to an init hook, afaics.

**Review by flotho — APPROVED** [review]

**Pull request merged**